h/services/user_delete.py

```diff
@@ -118,20 +118,31 @@ def __init__(self, db, job_queue, worker):
     def delete_authtickets(self, user):
         """Delete all AuthTicket's belonging to `user`."""
         self.worker.delete(
-            AuthTicket, select(AuthTicket.id).where(AuthTicket.user == user)
+            f"Deleted auth tickets belonging to {user!r}",
```

Comment:
`LimitedWorker.delete()` how has a required `log_message` argument that the caller must supply. This now logs _which user's_ auth tickets are being deleted, and `LimitedWorker.delete()` will handle logging the deleted auth tickets themselves. Using `!r` so that it logs `User.__repr__()`